libs/mobileapp/components/src/lib/card-product/card-product.tsx

```diff
@@ -12,50 +12,45 @@ type CardProductProps = {
   productLocation: string;
   productImage: string;
   productSeller: string;
-  padding?: string;
   productId: number;
+  productLabel: string;
 };
 
 export function CardProduct({
   productName,
   productPrice,
   productLocation,
   productImage,
-  productSeller,
   productId,
-  padding,
+  productLabel,
 }: CardProductProps) {
   function numberWithCommas(x: number | string): string {
     return x.toString().replace(/(\d)(?=(\d\d\d)+(?!\d))/g, '$1,');
   }
 
   return (
     <IonCard
-      style={{ textAlign: 'left', margin: padding ? padding : '0' }}
       href={`/product-detail/${productId}`}
+      className="cardProduct ion-text-left ion-no-padding"
```

Comment:
Ini dan css yg related lebih baik di samain aja deh mas @joshuanatanielnm untuk style penulisannya, karena yang lainnya pake `kebab-case`. mending ini dijadikan:

```suggestion
      className="card-product ion-text-left ion-no-padding"
```

Tujuannya selain biar seragam, juga pencarian className ini di editor nantinya lebih mudah mas.

Reply:
Ohhh iya mas kelupaan, thanks mas

Reply:
updated mas @rohmad-st